### PR TITLE
ADD community project pipeline

### LINKED ISSUE #721: Add pipeline training and inference Patchcore UI
I have already build a pipeline for training and inferencing Patchcore model on website. I want to contribute my pipeline for everyone can use it to quickly demo. Where folder can I create pull request in anomalib repo?

My pipeline Github: https://github.com/vnk8071/anomaly-detection-in-industry-manufacturing

With training:
- User can add custom dataset from zip file with MVTec format
- I use default backbone Resnet 18 and Patchcore model (Can be add more backbones and models)
- I created 2 custom datasets for training.

With inference:
- I have already training in MVTec dataset with 3 categories: Hazelnut, Metal Nut and Grid.
- 3 outputs: Anomaly score, target label and show heatmap on UI

I hope my pipeline can contribute to anomalib open-source.

### PR DESCRIPTION
**Pull request by vnk8071**
# Description

- Add community project about pipeline training and inference on website UI.

- Fixes #721

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which refactors the code base)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist

- [x] My code follows the [pre-commit style and check guidelines](https://github.com/openvinotoolkit/anomalib/blob/main/CONTRIBUTING.md) of this project.
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,6 +299,12 @@ For more information, refer to the [Logging Documentation](https://openvinotoolk
 
 Note: Set your API Key for [Comet.ml](https://www.comet.com/signup?utm_source=anomalib&utm_medium=referral) via `comet_ml.init()` in interactive python or simply run `export COMET_API_KEY=<Your API Key>`
 
+# Community Projects
+## 1. Pipeline training and inference on website
+In this project, I create a pipeline for training and inference (my PR #729) with all datasets format MVTec (custom) on UI. Everyone can training with all Anomalib models supported and quickly to demo. And my project is an end-to-end example in anomaly detection task. 
+
+For more details, everyone can visit and use my project below link. <p><a href="https://github.com/openvinotoolkit/anomalib/discussions/733">Link Show and tell</a> | <a href="https://github.com/vnk8071/anomaly-detection-in-industry-manufacturing/tree/master/anomalib_contribute">Github anomalib vnk8071</a></p>
+
```
Comment (ashwinvaidya17) [inline README.md:307]: Minor suggestion for edits.

```suggestion
# Community Projects
## 1. Web-based Pipeline for Training and Inference
This project showcases an end-to-end training and inference pipeline build on top of Anomalib. It provides a web-based UI for uploading MVTec style datasets and training them on the available Anomalib models. It also has sections for calling inference on individual images as well as listing all the images with their predictions in the database.

You can view the project on [Github](https://github.com/vnk8071/anomaly-detection-in-industry-manufacturing/tree/master/anomalib_contribute)
For more details see the [Discussion forum](https://github.com/openvinotoolkit/anomalib/discussions/733)
```

Comment (vnk8071) [reply README.md]: @ashwinvaidya17 Thanks for your suggestion and I have already change the commit.

**Review by ashwinvaidya17 — APPROVED** [review]
Again, fantastic work on the pipeline! Thanks for your efforts

**Review by djdameln — APPROVED** [review]
Thanks!

**Comment by ashwinvaidya17** [comment]
@vnk8071 Looks like the pre-commit checks have failed. Can you install pre-commit in your system as described here https://openvinotoolkit.github.io/anomalib/developer_guide/pre_commit_hooks.html. Then run `pre-commit run --all` and commit the modified files with the changes. Lemme know if something is not clear.

**Comment by ashwinvaidya17** [comment]
There seems to be some unintentional files committed to this PR. I can see some `.py` files in the notebooks folder.

**Comment by vnk8071** [comment]
@ashwinvaidya17 Thanks for your supported and I will close this PR.

**Pull request closed without merging**